### PR TITLE
use celery task to get json tree data

### PR DESCRIPTION
**Pull request by ivanistheone**
Getting the json data from Kolibri Studio takes a long time and times out.

This was causing 500 errors at the end of sushichef runs, and causing confusion for ricecooker users (the run succeeded and everything is fine on Studio, but the error 500 from sushibar makes it look like something went wrong).

The solution is to fetch json tree data in as a background task (celery worker).

### CODE REVIEW

File: config/settings/base.py [hunk 1]
```diff
@@ -314,3 +314,14 @@ def get_env(var, default=None):
...
 
+
+
+# Celery settings
+# ------------------------------------------------------------------------------
+CELERY_ACCEPT_CONTENT = ['application/json']
+CELERY_TASK_SERIALIZER = 'json'
+CELERY_RESULT_SERIALIZER = 'json'
+# the following values will be overwritren in production settings
+BROKER_URL = 'redis://localhost:6379/6'
```
Comment (aronasorman) [inline config/settings/base.py]: In production, do we have redis started in the same machine as the app itself?

File: config/settings/base.py [hunk 2]
```diff
@@ -314,3 +314,14 @@ def get_env(var, default=None):
...
+CELERY_RESULT_BACKEND = 'redis://localhost:6379/6'
```
Comment (aronasorman) [inline config/settings/base.py]: How did you determine to use the `6`th database here?

File: config/settings/production.py [hunk 3]
```diff
@@ -202,12 +202,27 @@
...
+    hostname=os.getenv("CELERY_BROKER_ENDPOINT"),
+    port=os.getenv("CELERY_REDIS_PORT"),
+    db=os.getenv("CELERY_REDIS_DB")
+) or BROKER_URL
+CELERY_RESULT_BACKEND = "redis://{hostname}:{port}/{db}".format(
+    hostname=os.getenv("CELERY_RESULT_BACKEND_ENDPOINT"),
+    port=os.getenv("CELERY_REDIS_PORT"),
+    db=os.getenv("CELERY_REDIS_DB")
+) or CELERY_RESULT_BACKEND
+CELERY_TIMEZONE = os.getenv("CELERY_TIMEZONE") or CELERY_TIMEZONE
```
Comment (aronasorman) [inline config/settings/production.py]: Nice! Straight from Studio :D

File: config/settings/base.py (hunk 2 above)
Comment (ivanistheone) [reply config/settings/base.py]: 0 and 1 are used to store "chef progress state" so I picked 5 and 6 as random, different numbers higher than 0 and 1.

**Pull request merged**